app/src/main/java/com/spoony/spoony/presentation/placeDetail/PlaceDetailRoute.kt

```diff
@@ -0,0 +1,310 @@
+                )
+
+                Spacer(modifier = Modifier.height(8.dp))
+
+                Text(
+                    text = title,
+                    style = SpoonyAndroidTheme.typography.title1,
+                    color = SpoonyAndroidTheme.colors.black
+                )
+
+                Spacer(modifier = Modifier.height(8.dp))
+
+                Text(
+                    text = date,
+                    style = SpoonyAndroidTheme.typography.caption1m,
+                    color = SpoonyAndroidTheme.colors.gray400
+                )
+
+                Spacer(modifier = Modifier.height(20.dp))
+
+                Text(
+                    text = description,
+                    style = SpoonyAndroidTheme.typography.body2m,
+                    color = SpoonyAndroidTheme.colors.gray900
+                )
+
+                Spacer(modifier = Modifier.height(32.dp))
+
+                StoreInfo(
+                    isBlurred = !isScoop,
+                    menuList = menuList,
+                    locationSubTitle = placeName,
+                    location = placeAddress
+                )
+            }
+            Spacer(modifier = Modifier.height(27.dp))
+        }
+        PlaceDetailBottomBar(
+            modifier = Modifier,
+            count = zzinCount,
+            isScoop = isScoop,
+            isZzim = isZzim,
+            onSpoonEatClick = updateScoop,
+            onSearchMapClick = {
+                // 네이버 길찾기 코드
+            },
+            onMyMapSelectClick = updateZzim
+        )
+    }
+}
+
+@Composable
+fun PlaceDetailBottomBar(
+    count: Int,
+    onSpoonEatClick: (Boolean) -> Unit,
+    onSearchMapClick: () -> Unit,
+    onMyMapSelectClick: (Boolean) -> Unit,
+    modifier: Modifier = Modifier,
+    isScoop: Boolean = false,
+    isZzim: Boolean = false
+) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .background(SpoonyAndroidTheme.colors.white)
+            .padding(
+                horizontal = 20.dp,
+                vertical = 10.dp
+            ),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        if (isScoop) {
+            SpoonyButton(
+                text = "길찾기",
+                onClick = onSearchMapClick,
+                style = ButtonStyle.Secondary,
+                size = ButtonSize.Medium,
+                modifier = Modifier.weight(1f)
+            )
+
+            Spacer(modifier = Modifier.width(15.dp))
+
+            Column(
+                modifier = Modifier
+                    .width(56.dp)
+                    .noRippleClickable { onMyMapSelectClick(isZzim) },
+                verticalArrangement = Arrangement.Center,
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+                Icon(
+                    imageVector = if (!isZzim) ImageVector.vectorResource(id = R.drawable.ic_add_map_gray400_24) else ImageVector.vectorResource(id = R.drawable.ic_add_map_main400_24),
```

Comment:
```suggestion
                    imageVector = if (isZzim) ImageVector.vectorResource(id = R.drawable.ic_add_map_gray400_24) else ImageVector.vectorResource(id = R.drawable.ic_add_map_main400_24),
```
P4: 저는 이렇게 하는걸 더 선호합니다..ㅎㅎㅎㅎ

Reply:
수정했습니다!